flopy/mf6/mfmodel.py

```diff
@@ -114,7 +114,7 @@ def __init__(
         verbose=False,
         **kwargs
     ):
-        super().__init__(simulation.simulation_data, modelname)
+        super(MFModel, self).__init__(simulation.simulation_data, modelname)
```

Comment:
there are ~12 of these that could also be restored to use just `super()` (see #1082)

Reply:
@mwtoews Updated.